app/components/explosions/explosions.html.erb

```diff
@@ -0,0 +1,14 @@
+<section class="grid">
+  <div class="line">
+  <% (0..@size*@size-1).each_with_index do |coord, i| %>
+    <% if i % @size == 0 && i != 0 %>
```

Comment:
similar comment here. this could be a predicate method that takes `i` as an argument. one of the big benefits of the view component pattern is the ability to write tests for logic that directly impacts the views. there's some cool and complex logic in this view and my personal preference is to read it in a Ruby class vs an ERB template :)